src/Form/Type/FormatterType.php

```diff
@@ -44,9 +45,9 @@ final class FormatterType extends AbstractType
     protected $translator;
 
     /**
-     * @var ConfigManagerInterface
+     * @var ConfigManagerInterface|CKEditorConfigurationInterface
```

Comment:
```suggestion
     * @deprecated since sonata-project/formatter bundle 4.x and will be removed in version 5.0.
     * 
     * @var ConfigManagerInterface
```